redash/query_runner/elasticsearch.py

```diff
@@ -252,4 +285,85 @@ def run_query(self, query):
         return json_data, error
 
 
+#
+#
+# Simple query example:
+#
+# - Query the index named "twitter"
+# - Filter by user equal "kimchy"
+# - Return the fields: "@timestamp", "tweet" and "user"
+# - Return up to 15 results
+# - Sort by @timestamp ascending
+#
+# {
+#     "index" : "twitter",
+#     "query" : {
+#       "match": {
+#           "user" : "kimchy"
+#       }
+#     },
+#     "fields" : ["@timestamp", "tweet", "user"],
+#     "size" : 15,
+#     "sort" : "@timestamp:asc"
+# }
+#
+#
```

Comment:
Worth moving this to the doc page: http://docs.redash.io/en/latest/usage/elasticsearch_querying.html